pkg/plugins/resources/k8s/native/config/crd/bases/kuma.io_circuitbreakers.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Can we remove this entire `pkg/plugins/resources/k8s/native` directory as well? I think it's confusing to have a kubebuilder operator that is not used.

Reply:
Tried it, but not that easy though. Maybe we have to do it in a dedicated PR.

Reply:
Sad, thanks for trying.